### PR TITLE
Added basic structure for importing elements

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

**Comment by branflake2267** [comment]
Cool!

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                        Imports imports = GWT.create(Imports.class);
+                        imports.injectImports();
+                        doCallback(callback);
+                    }
+                }).inject();
+    }
+
+    private static void doCallback(final LoadCallback callback) {
+        if (callback != null) {
+            Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {
```
Comment (Chris-V) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java]: Is the scheduler necessary?

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/rebind/RebindModule.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    @Provides
+    @Singleton
+    VelocityEngine getVelocityEngine(
+            @Named("VELOCITY_PROPERTIES") String velocityProperties,
+            TreeLogger logger)
+            throws UnableToCompleteException {
+        try {
+            InputStream inputStream = null;
+            Properties properties = new Properties();
+            try {
```
Comment (Chris-V) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/rebind/RebindModule.java]: Could be a try-with-resource

**Comment by Chris-V** [comment]
LGTM

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/GwtPolymer.gwt.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit 2.7.0//EN"
+        "https://raw.githubusercontent.com/gwtproject/gwt/2.7.0/distro-source/core/src/gwt-module.dtd">
+<module>
+    <inherits name="com.arcbees.gwtpolymer.elements.webcomponentsjs.Webcomponentsjs"/>
```
Comment (Chris-V) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/GwtPolymer.gwt.xml:8]: Remove? Seems like one of the generated modules and it's empty.

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webapp",
+  "version": "0.0.0",
+  "homepage": "https://github.com/ArcBees/gwt-polymer",
+  "authors": [
+    "Maxime Mériouma-Caron <maxime.meriouma@arcbees.com>"
+  ],
+  "license": "MIT",
```
Comment (Chris-V) [inline bower.json:8]: The other files and parent pom are under Apache 2.0

**Comment by christiangoudreau** [comment]
Guys, we need to move forward... why isn't this merged yet?

**Comment by meriouma** [comment]
Trying to take the time to finish something for this. Will try to do this ASAP..

**Pull request merged**